### ISSUE TITLE
Warning: Failed child... type number, expected string (ActionSheet)

### ISSUE THREAD

**Issue by jacargentina** [GeekyAnts/NativeBase#1571]
## react-native, react and native-base version
react-native 0.53
react 16.2.0
native-base 2.3.8

## Expected behaviour
Work just fine without warnings (on Debug)
## Actual behaviour
When activating the ActionSheet, the warning its displayed.
## Steps to reproduce (code snippet or screenshot)
<img width="379" alt="captura de pantalla 2018-02-06 a la s 11 31 50" src="https://user-images.githubusercontent.com/517228/35864746-5e7f0b78-0b31-11e8-8c0a-4af2d2d63636.png">

## Is the bug present in both ios and android or in any one of them?
both platforms. iOS AND android
## Any other additional info which would help us debug the issue quicker.

On the ActionSheet.js file, the keyExtractor returns the index, but it should return it converted to string 
```
            <FlatList
              style={{ marginHorizontal: -15, marginTop: 15 }}
              data={this.state.items}
              keyExtractor={(item, index) => index}
```

should look:
```
            <FlatList
              style={{ marginHorizontal: -15, marginTop: 15 }}
              data={this.state.items}
              keyExtractor={(item, index) => String(index) }
```

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.3.9](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.3.9)

**Comment by andfs** [comment]
The same is happening on Picker in ios

**Comment by flindenberg** [comment]
We're experiencing this with the Picker on iOS, too.

**Comment by CristiTr** [comment]
The problem is unresolved on iOS in the Picker
 "native-base": "2.3.9",
 "react": "16.2.0",
 "react-native": "0.53.3",

**Comment by akhil-geekyants** [comment]
@flindenberg @CristiTr this will be fixed in upcoming release

**Comment by flindenberg** [comment]
@akhil-geekyants, that's great news. Thank you.

**Comment by SupriyaKalghatgi** [comment]
Fixed with NB [2.3.10](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.3.10)

**Comment by CristiTr** [comment]
Thanks! I can confirm that this issue was resolved.
